incubator/helm-release-pruner/templates/configmap.yml

```diff
@@ -6,110 +6,6 @@ data:
   start-up.sh: |-
     #!/bin/sh
 
-    apk add --update bash coreutils
-
 {{- range .Values.pruneProfiles }}
     /usr/local/bin/prune-releases.sh --older-than="{{ .olderThan }}" --helm-release-filter="{{ .helmReleaseFilter }}" --namespace-filter="{{ .namespaceFilter }}"{{ if $.Values.job.dryRun }} --dry-run{{ end }}
```

Comment:
In general I think we should make this script the entrypoint for the container and just skip start script if we're going to do it with a custom image.

Would be nice to make all the parameters ENV for the new container. Not blocking for this PR

Reply:
Yeah that definitely gets complicated because of the chart having a variable length loop. The loop params either have to explicitly generate the loop steps as they do now via helm or they would have to be passed into the container which could get messy with 3 parameters per iteration (comma separated values?).